app/src/main/java/com/github/h3lp3rs/h3lp/locationmanager/LocationHelper.kt

```diff
@@ -0,0 +1,53 @@
+import android.content.Context
+import android.content.Intent
+import android.location.Location
+import android.location.LocationManager
+import com.github.h3lp3rs.h3lp.MainPageActivity
+import com.github.h3lp3rs.h3lp.locationmanager.GeneralLocationManager
+
+
+/**
+ * SuperActivity that is common to all activities using localization
+ */
+class LocationHelper {
+    private var userLocation: Location? = null
```

Comment:
By convention I wasn't a fan of public attributes that would allow an activity to modify the location, that means we need the getters anyway. Consider this point, how we store the location internally is just a detail of implementation, but your right that storing two doubles would be possible.